k2/csrc/fsa_utils.cu

```diff
@@ -110,8 +110,7 @@ static void SplitStringToVector(const std::string &in, const char *delim,
 
 /* Create an acceptor from a stream, assuming the acceptor is in the k2 format:
 
-   src_state1 dest_state1 label1 score1
-   src_state2 dest_state2 label2 score2
+   src_state dest_state label [aux_label1 aux_label2 ... ] score
```

Comment:
If `aux_label_x` is present, does it assume that every arc has the same number of `aux_labels`?

I guess so as the output argument is `Array2`, not `Ragged<int32_t>`.
It would be great if it is documented.

Reply:
.. this is kind of implicit because `num_aux_labels` is an arg to the function.